qiskit/providers/ibmq/jupyter/dashboard/watcher_monitor.py

```diff
@@ -54,35 +60,42 @@ def _job_checker(job: IBMQJob, status: JobStatus, watcher: 'IQXDashboard') -> No
             exception_count = 0
 
             if status.name == 'QUEUED':
-                queue_pos = job.queue_position()
-                if queue_pos != prev_queue_pos:
-                    queue_info = job.queue_info()
-                    if queue_info and queue_info.estimated_start_time:
-                        est_time = duration_difference(queue_info.estimated_start_time)
-                        prev_est_time = est_time
-                    else:
-                        est_time = prev_est_time
-
-                    update_info = (job.job_id(), status.name+' ({})'.format(queue_pos),
-                                   est_time, status.value)
-
-                    watcher.update_single_job(update_info)
-                    if queue_pos is not None:
-                        interval = max(queue_pos, 2)
-                    else:
-                        interval = 2
-                    prev_queue_pos = queue_pos
+                if job_type == JobType.IBMQ:
+                    queue_pos = job.queue_position()
+                    if queue_pos != prev_queue_pos:
+                        queue_info = job.queue_info()
+                        if queue_info and queue_info.estimated_start_time:
+                            est_time = duration_difference(queue_info.estimated_start_time)
+                            prev_est_time = est_time
+                        else:
+                            est_time = prev_est_time
+
+                        update_info = (job.job_id(), status.name+' ({})'.format(queue_pos),
+                                       est_time, status.value)
+
+                        watcher.update_single_job(update_info)
+                        if queue_pos is not None:
+                            interval = max(queue_pos, 2)
+                        else:
+                            interval = 2
+                        prev_queue_pos = queue_pos
+                else:
+                    update_info = (job.job_id(), status.name)
+                    # Update the job on the dashboard
+                    watcher.update_single_job(job_type, update_info)
 
             elif status.name != prev_status_name:
                 msg = status.name
                 if msg == 'RUNNING':
-                    job_mode = job.scheduling_mode()
+                    # Note: As of now, all Runtime jobs are considered priority jobs.
+                    job_mode = job.scheduling_mode() if job_type == JobType.IBMQ else 'PRIORITY'
```

Comment:
In theory runtime job can be in dedicated mode as well, but not information is not currently available. I think it's best to just not show any scheduling mode for them.

Reply:
fixed!